### DISCUSSION TITLE
Buffalo ls-wxl

### DISCUSSION THREAD

**Discussion by jhmc93** [1000001101000/Debian_on_Buffalo#179] [General]
Pretty simple really, will the Debian run on this model (Buffalo linkstation ls-wxl)

**Comment by 1000001101000** [comment]
Yes

**Reply by jhmc93** [reply]
Do I need certain image?

**Comment by 1000001101000** [comment]
Yes, the images have the model as part of the filename

**Reply by jhmc93** [reply]
Sorry what file name is that

**Comment by 1000001101000** [comment]
Read through everything in the wiki before you get too far:
https://github.com/1000001101000/Debian_on_Buffalo/wiki

It can be a pain to reset everything if you mess up a step

**Reply by jhmc93** [reply]
Ye it looks hard, too get it to work on that specific model
